conformity/fields/meta.py

```diff
@@ -8,6 +8,28 @@
 from conformity.utils import strip_none
 
 
+@attr.s
+class Nullable(Base):
+    """
+    Accepts the field type passed as the first positional argument or a value of null/None.
+    """
+
+    field = attr.ib()
+    description = attr.ib(default=None)
+
+    def errors(self, value):
+        if value is None:
+            return []
+
+        return self.field.errors(value)
+
+    def introspect(self):
+        return strip_none({
+            'type': 'nullable({})'.format(self.field.introspect),
```

Comment:
Please use double quotes for consistency with the rest of the library.

Reply:
Fair.